src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -2308,6 +2312,163 @@ In case of PAM users, the login file should not be modified as password authenti
+For example, if the role `proactive-workflow-engineer` is configured in Keycloak, the role/group to be created in proActive must be as follows:
+
+----
+grant principal org.ow2.proactive.authentication.principals.GroupNamePrincipal "proactive-workflow-engineer" {
+  // list of granted privileges
+}
+----
+
+===== Specific example of Keycloak configuration
+
+Below is an example of Keycloak configuration which provides sample values for the Keycloak properties to be configured in the file `config/authentication/keycloak.cfg`.
+[source,properties]
+----
+# URL of a Keycloak server configured for ProActive authentication
+keycloak.url=https://keycloak.activeeon.com:9443
+
+# When Keycloak URL uses the HTTPS protocol, this parameter indicates which kind of TLS certificate to consider for the Keycloak server.
+# True means that any certificate can be used including the self-signed ones.
+# False means that only valid certificates generated by trusted Certification Authorities are allowed.
+keycloak.https.allow_any_certificate=true
+
+# Keycloak realm in which ProActive client is configured
+keycloak.realm=activeeon
+
+# Keycloak client credentials (note that ProActive is the client application of Keycloak)
+keycloak.proactive.client_id=ProActive
+keycloak.proactive.client_secret=w9jvqPPTMvxxxyyyrnessKxt7nCs75NpDZ
+
+# Scope of the information to be returned by Keycloak for the ProActive client.
+# An empty value considers the scopes defined for ProActive client in Keycloak.
+keycloak.proactive.scope=
+
+# Keycloak RSA public key used to decrypt the access tokens
+keycloak.public.key=MIIBIjANBgkutvbS5w0BAQEFAAOCAQ8AMGGBCgKCAQEAsmVExiiZgtnrN7DrPRu6Cx/H1Fx4wOCy1b9sT4w7YjSLfX0d5GN51bDiz5olcvgBNvNIXGXm3Vax8OoOJdIYaq2EiVa0p0ATpzYFIZkKFOkiuriDJd3TpxuJyWIoEg1zUZHNPw0zvFftZ24HByZr51qm/QMBQNvj2tMcPvBgm5tUbIe1bzR3VURXA41+p0Cs4I1kjhWV5B95+SLbxcj/+TMkUt5BVDxPy0nJixM+4zZ/R48OCTpPUnLrGJw+/z5vgbqwwH+ekpI+L08v0wUYmMkqfnykZxOyhtoCUD66pojDoPO60JPDj5qtbiydHwV7DsULvbc406wEe0pN8hKO5QIDAQAB
+
+# The attribute in Keycloak access token to be used to extract the user/principal name
+keycloak.principal_attribute=name
+
+# Use the resource_access block in Keycloak access token to extract the user/principal roles
+keycloak.use_resource_role_mappings=true
+
+# Authentication fallback Properties
+# check user password and group membership in login and group files
+# (as performed in FileLogin method), if user is not found in Keycloak.
+#
+# It use pa.[scheduler.core|RM].defaultloginfilename to authenticate user and
+# pa.[scheduler.core|RM].defaultgroupfilename files to check user membership.
+# In any case, no fallback is performed if user is found in Keycloak but has entered an incorrect password).
+# true or false.
+pa.keycloak.authentication.fallback=false
+
+# check user group membership in group file (as performed in FileLogin method)
+# if user is not found in its corresponding requested Keycloak group.
+# true or false.
+pa.keycloak.group.membership.fallback=false
```

Comment:
should at least be set to true in the example configuration

Reply:
Done.